CHANGES

```diff
@@ -326,15 +326,15 @@ Version 1.1.4 (2015-03-22)
 * add confirm dialog before restoring (https://launchpad.net/bugs/438079)
 * Fix bug: Wrong status text in the tray icon (https://launchpad.net/bugs/1429400)
 * add option to run only one snapshot at a time
-* Fix bug: restore permissions of lots of files made BackInTime unresposive (https://launchpad.net/bugs/1428423)
+* Fix bug: restore permissions of lots of files made BackInTime unresponsive (https://launchpad.net/bugs/1428423)
 * Fix bug: failed to restore file owner and group
-* cache uuid in config so it doesn't fail if the device isn't pluged in (https://launchpad.net/bugs/1426881)
+* cache uuid in config so it doesn't fail if the device isn't plugged in (https://launchpad.net/bugs/1426881)
 * add warning about wrong Python version in configure
 * prevent snapshots from being removed with restore and delete; show warning if restore and delete filesystem root (https://answers.launchpad.net/questions/262837)
 * Fix bug: OSError in free_space; add alternate method to get free space
 * add bash-completion
 * Fix bug: ugly theme while running as root on Gnome based DEs (https://launchpad.net/bugs/1418447)
-* Fix bug: filename with broken charset throwed UnicodeError exception (https://launchpad.net/bugs/1419694)
+* Fix bug: filename with broken charset trowed UnicodeError exception (https://launchpad.net/bugs/1419694)
```

Comment:
That is not correct.
I assume you used the auto-correct feature of codespell? Using tools that manipulate the code automatic (e.g. `black`) need to be mentioned when opening PRs and is not recommended.

Reply:
@buhtz I did changes manually, major was spotted by codespell, after I opened the files with errors and fixed them excluding the cases where are code, other languages or other things to not changes. In the editor I have also an extension which underlines words that are not in the English vocabulary, most are not to be considered while some caught my eye because they were actually errors.
others was related to function name, for example:
`SotedcomoBox, SnapshotComba, ProfilCombo` for each I searched the function with correct name in the file and I corrected them, for example to: `SortedcomboBox, SnapshotCombo, ProfileCombo`
regarding the changelog, probably because I was already going through and checking files for an hour, it hadn't occurred to me that it could refer to "Python Throw Exception", I think I made a mistake, sorry, I hope I haven't made any other mistakes
I hope I haven't made any other mistakes also due to my imperfect English^^'

Reply:
I really appreciate your work. You have invested a lot of time. And I apologize if I might have become a bit harsh.
It is easy to drive me emotional when it comes to Microsoft, Ubuntu or something similar. ;) And please be aware that the "BIT Team" currently have not official consents about things like this. Most of the time it is my personal opinion.

Please fix the "thrown" typo and remove the workflow-codespell file from your PR. Then we can merge it.

I will add codespell on my todo list and see if it make sense to integrate it into our unittests (like we do with PyLint in #1562). It might give to much false positives.